src/components/basic-element/basic-element.jsx

```diff
@@ -226,11 +223,17 @@ var BasicElement = React.createClass({
    * if they'd otherwise run off the page.
    */
   handleTranslation: function(x, y) {
-    var dl = this.dragLimits,
-        w_edge = dl.width,
-        h_edge = dl.height;
+
+    var a = this.state.angle;
```

Comment:
`a` is pretty non-semantic. Could we just do `angle` or something more descriptive?